python_cmake_module/package.xml

```diff
@@ -10,7 +10,7 @@
 
   <buildtool_depend>ament_cmake</buildtool_depend>
 
-  <buildtool_export_depend>python3-dev</buildtool_export_depend>
+  <buildtool_export_depend>python3</buildtool_export_depend>
```

Comment:
@dirk-thomas this change is somewhat different from the others. The dependency key for python3-dev was already present from https://github.com/ros2/rosidl/commit/1d7b4eb15a0fae579ec4e02f1345156886235200. All I did was normalize the keyname to match the rosdep key rather than the ubuntu package name.

If you don't think it should be there at all, I think that is a separate change to be discussed. Build dependencies for ament_python packages will be added as needed but this is an ament_cmake package and if it needs python at build time, or needs to export a dependency on python at build time, this must remain.